Docs/install/install_host.rst

```diff
@@ -36,21 +36,35 @@
 
 .. _installation-host:
 
-##############################
-AIMET Installation and Setup
-##############################
+###################################
+AIMET manual installation and setup
+###################################
 
-This page provides instructions to install AIMET package on Ubuntu 22.04 LTS with Nvidia GPU. Please follow the instructions in the order provided, unless specified otherwise.
+This page describes how to manually install AIMET, including all prerequisites and dependencies, for all framework and GPU variants.
```

Comment:
Can we just say `for all frameworks and variants` , since this page covers CPU variants as well?

Reply:
Changed.